src/progpy/loading/gaussian_wrapper.py

```diff
@@ -22,6 +22,10 @@ class GaussianNoiseLoadWrapper():
     -------------------
     seed: {int, SeedSequence, BitGenerator, Generator}, optional
         The seed for random number generator. This can be set to make results repeatable.
+    std_slope: float, optional
+        The increase in standard deviation per second of time. Std is the standard deviation at t<=t0.
+    t0: float, optional
```

Comment:
For consistency, should this be called `std_t0`?

Reply:
It depends- t0 is more consistent with t, the parameter that it's compared against, but std_t0 would be consistent with std/std_slope. I think I'd like to keep t0